README.md

```diff
@@ -178,6 +185,24 @@ Pwwka.configure do |config|
 end
 ```
 
+#### Sending message Async with Sidekiq
+
+To enqueue a message in a background Sidekiq job, use `Transmitter.send_message_async_sidekiq`
+```ruby
+Pwwka::Transmitter.send_message_async_sidekiq(payload, routing_key, delay_by_ms: 5000) # default delay is 0
```

Comment:
Should we instead overload `send_message_async` to have like `sidekiq: true`, or `queue_with: :sidekq`?

Or, maybe have a config option like `background_job_processor: ` that defaults to resque and that one could set to sidekiq, which would change the behavior of `send_message_async` but also configure the proper background job?

Reply:
I'm not sure if this is gross or not, but maybe we could inspect the configured `async_job_klass` to know how to handle it?

Reply:
yeah, true.  Or have the async job class adopt a protocol i.e. method that queues a job to itself?

Reply:
I updated this PR to try out the overloading `.send_message_async` method. what do you all think?